indigo_za/refs.py

```diff
@@ -10,16 +10,23 @@ class RefsFinderENG(BaseRefsFinder):
 
         Act 52 of 2001
         Act no. 52 of 1998
+        Constitution [of [the Republic of] South Africa] [Act][,] [1996]
+
     """
 
     # country, language, locality
     locale = (None, 'eng', None)
 
-    act_re = re.compile(r'Act,?\s+(no\.?\s*)?(\d+)+\s+of\s+(\d{4})', re.I)
-    candidate_xpath = ".//text()[contains(., 'Act') and not(ancestor::a:ref)]"
+    act_re = re.compile(r'Act,?\s+([nN]o\.?\s*)?(\d+)+\s+of\s+(\d{4})|\bConstitution\b(\s+of(\s+the\s+Republic\s+of)?\s+South\s+Africa)?((\s+Act)?,?\s+1996)?')
```

Comment:
Is the second `\b` in  `\bConstitution\b` necessary?
Maybe we should use `\b` for `r'\bAct`, too.

Reply:
I think for Act adding a `\b` at the start would prevent it from picking up 'fAct 20 of 2015', but what are the odds?

Reply:
I think the first is necessary (and Act should have it too, that's an oversight). I understand the second one now -- the alternative is to ensure it has whitespace afterwards, but this is better.